linux_os/guide/system/logging/journald/service_systemd-journald_enabled/rule.yml

```diff
@@ -0,0 +1,41 @@
+documentation_complete: true
+
+title: 'Enable systemd-journald Service'
+
+description: |-
+    The <tt>systemd-journald</tt> service is an essential component of
+    systemd.
+    {{{ describe_service_enable(service="systemd-journald") }}}
+
+rationale: |-
+    In the event of a system failure, {{{ full_name }}} must preserve any information necessary to determine cause of failure and any information necessary to return to operations with least disruption to system processes.
+severity: medium
+
+
+identifiers:
+    cce@rhel8: CCE-85921-5
```

Comment:
We're going to use this new rule in RHEL 9 STIG, so we should add also a RHEL 9 CCE.